basic/dapps/universal-provider-solana/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sandbox",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
+    "@solana/web3.js": "^1.78.4",
+    "@walletconnect/modal": "^2.6.1",
+    "@walletconnect/universal-provider": "2.10.0-canary-ac4c97",
```

Comment:
Can you elaborate the use of the canary here?

Reply:
good catch, updating it tonight, should be `2.10.0`

Reply:
Bump